<launch>

<rosparam param="base_width">0.5</rosparam>
<rosparam param="ticks_meter">0.06 </rosparam>
<rosparam param="map_vel">0.3 </rosparam>

<node pkg="beginner_tutorials" type="joy2twist" name="joy2twist" output="screen"/>


<!--node name="base_node" pkg="beginner_tutorials" type="base_node.py"-->
<!--rosparam file="$(find beginner_tutorials)/param/board_base_config.yaml" command="load" /-->
<!--/node-->



<node pkg="beginner_tutorials" type="vel_ctrl" name="l_vel" output="screen">

<remap from ="lec" to="left_lec"/> 
<remap from ="offset" to="left_offset"/> 
<remap from ="pos_des" to="left_vel_des"/>

<remap from ="ang" to="left_pos"/> 
<remap from ="vel" to="left_vel"/> 
<remap from ="out" to="left_out"/>

<rosparam param ="kp_vel">30</rosparam>
<rosparam param ="ki_vel">0.1</rosparam>
<rosparam param ="kd_vel">0</rosparam>
<rosparam param ="km_vel">20</rosparam>
<rosparam param ="umbral_vel">0.2</rosparam>
<rosparam param ="range_vel">150</rosparam>
<rosparam param ="kierr_vel">0.2</rosparam>
<rosparam param ="kimax_vel">100</rosparam>
<rosparam param ="kisum_vel">0</rosparam>

</node>

<node pkg="beginner_tutorials" type="vel_ctrl" name="r_vel" output="screen">

<remap from ="lec" to="right_lec"/> 
<remap from ="offset" to="right_offset"/> 
<remap from ="pos_des" to="right_vel_des"/>

<remap from ="ang" to="right_pos"/> 
<remap from ="vel" to="right_vel"/> 
<remap from ="out" to="right_out"/>

<rosparam param ="kp_vel">30</rosparam>
<rosparam param ="ki_vel">0.1</rosparam>
<rosparam param ="kd_vel">0</rosparam>
<rosparam param ="km_vel">20</rosparam>
<rosparam param ="umbral_vel">0.2</rosparam>
<rosparam param ="range_vel">150</rosparam>
<rosparam param ="kierr_vel">0.2</rosparam>
<rosparam param ="kimax_vel">100</rosparam>
<rosparam param ="kisum_vel">0</rosparam>

</node>


<node pkg="beginner_tutorials" type="twist2motor" name="twist2motor" output="screen"/>

<!--node pkg="beginner_tutorials" type="diff_odom" name="diff_odom" output="screen"/--> 

</launch>
